### ISSUE TITLE
How to get model pred_scores？

### ISSUE THREAD

**Issue by loyal-ikun** [openvinotoolkit/anomalib#818]
I want to get model output, such as the prediction score of the model. When I run the python test.py --config xx,  I only get the metric of model, but not the prediction of dataset.
I also try other method that load trained dataset and mode to predict output. But there are always bug in the sklearn.metric module. Besides I don't know how to call adaptive score calculation on non PL framework code.
I had to turn to the authors to solve the above problems.

**Comment by thanh-repo** [comment]
Hi, try to get the output of `trainer.test` command in `test.py` script
https://github.com/openvinotoolkit/anomalib/blob/c82ddcd76bed47d196cffbee333a995e7f1df3ed/tools/test.py#L49

It should be return some information might you need: 
https://github.com/Lightning-AI/lightning/blob/ca88f813a440bef61e611dc3c40343a2774cd21a/src/pytorch_lightning/trainer/trainer.py#L769-L773

**Comment by loyal-ikun** [comment]
@thanh-repo I have seen the link you gave me. It says that test can only return metric result, but I need the prediction of the model.
